react/Rating/Rating.demo.js

```diff
@@ -19,6 +33,20 @@ export default {
             rating
           })
         }))
+    },
+    {
+      label: 'Size',
+      type: 'radio',
+      states: ['substandard', 'superstandard', 'heading']
```

Comment:
These should be title case, and `Standard` should probably be the first option, removing `substandard` from the initial props.